code/modules/events/money_spam.dm

```diff
@@ -98,7 +100,7 @@
 					sender = pick("Dr","Crown prince","King Regent","Professor","Captain")
 					sender += " " + pick("Robert","Alfred","Josephat","Kingsley","Sehi","Zbahi")
 					sender += " " + pick("Mugawe","Nkem","Gbatokwia","Nchekwube","Ndim","Ndubisi")
-					message = pick("YOUR FUND HAS BEEN MOVED TO [pick("Sol","Biesel","Dominia","Elyra","Qerrbalak","Adhomai","NTCC Odin","Hephaestus","Kentucky")] DEVELOPMENTARY BANK FOR ONWARD REMITTANCE.",\
+					message = pick("YOUR FUND HAS BEEN MOVED TO [pick("Sol","Biesel","Dominia","Elyra","Qerrbalak","Adhomai","NTCC Odin","Hephaestus","Chicago")] DEVELOPMENTARY BANK FOR ONWARD REMITTANCE.",\
```

Comment:
Should change that to something that is common in our universe.

Reply:
Handled